diff --git a/config_generator/capabilities_reader.py b/config_generator/capabilities_reader.py
index 0dac9ed..1e6327f 100644
--- a/config_generator/capabilities_reader.py
+++ b/config_generator/capabilities_reader.py
@@ -1,12 +1,11 @@
 from collections import OrderedDict
-import json
 from urllib.parse import urljoin, urlparse
 from xml.etree import ElementTree
 
 import requests
 import os
 from pathlib import Path
-from shutil import move, copyfile
+from shutil import copyfile
 
 
 class CapabilitiesReader():
@@ -148,15 +147,15 @@ class CapabilitiesReader():
                 has_default = True
 
         # This is needed because we don't want to
-        # print the error message "thumbmail dir not found"
+        # print the error message "thumbnail dir not found"
         # multiple times
         thumbnail_dir_exists = True
         thumbnail_directory = ""
         if self.qwc_base_dir is None:
             thumbnail_dir_exists = False
             self.logger.info(
-                            "Skipping automatic thumbnail search "
-                            "(qwc2_base_dir was not set)")
+                "Skipping automatic thumbnail search "
+                "(qwc2_base_dir was not set)")
         else:
             thumbnail_directory = os.path.join(
                 self.qwc_base_dir, "assets/img/mapthumbs")
@@ -168,7 +167,7 @@ class CapabilitiesReader():
                     fname = os.path.join(dirpath, filename)
                     relpath = os.path.relpath(dirpath,
                                               qgis_projects_base_dir)
-                    wmspath = os.path.join(relpath, Path(filename).stem)
+                    wmspath = os.path.normpath(os.path.join(relpath, Path(filename).stem))
                     wmsurlpath = urlparse(urljoin(self.default_qgis_server_url, wmspath)).path
 
                     # Add to themes items
@@ -195,7 +194,7 @@ class CapabilitiesReader():
                         self.logger.info("Scanning for thumbnail(%s) under %s" % (
                             thumbnail_filename, thumbnail_directory))
                         thumbnail_path = os.path.join(
-                                thumbnail_directory, thumbnail_filename)
+                            thumbnail_directory, thumbnail_filename)
 
                         if os.path.exists(thumbnail_path):
                             self.logger.info("Thumbnail: %s was found" % (
