diff --git a/server.py b/server.py
index 05ec05e..708b09d 100644
--- a/server.py
+++ b/server.py
@@ -33,8 +33,8 @@ app.config['JWT_ACCESS_TOKEN_EXPIRES'] = int(os.environ.get(
 jwt = jwt_manager(app)
 app.secret_key = app.config['JWT_SECRET_KEY']
 
-i18n.set('load_path', [
-    os.path.join(os.path.dirname(__file__), 'translations')])
+i18n.set('load_path', [os.path.join(os.path.dirname(__file__), 'translations')])
+i18n.set('file_format', 'json')
 SUPPORTED_LANGUAGES = ['en', 'de']
 # *Enable* WTForms built-in messages translation
 # https://wtforms.readthedocs.io/en/2.3.x/i18n/
