diff -ur qca-2.3.8/plugins/qca-botan/CMakeLists.txt ../qca-2.3.8/plugins/qca-botan/CMakeLists.txt
--- qca-2.3.8/plugins/qca-botan/CMakeLists.txt	2024-01-09 12:12:58.000000000 +0100
+++ ../qca-2.3.8/plugins/qca-botan/CMakeLists.txt	2024-03-14 05:00:13.579783400 +0100
@@ -1,12 +1,14 @@
-find_package(PkgConfig REQUIRED)
+find_package(PkgConfig)
 
-pkg_check_modules(BOTAN IMPORTED_TARGET botan-2)
-if(NOT BOTAN_FOUND)
-  pkg_check_modules(BOTAN IMPORTED_TARGET botan-3)
-  if(BOTAN_FOUND)
-    set(CMAKE_CXX_STANDARD 20)
-    set(CMAKE_CXX_STANDARD_REQUIRED ON)
-    set(CMAKE_CXX_EXTENSIONS OFF)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(BOTAN IMPORTED_TARGET botan-2)
+  if(NOT BOTAN_FOUND)
+    pkg_check_modules(BOTAN IMPORTED_TARGET botan-3)
+    if(BOTAN_FOUND)
+      set(CMAKE_CXX_STANDARD 20)
+      set(CMAKE_CXX_STANDARD_REQUIRED ON)
+      set(CMAKE_CXX_EXTENSIONS OFF)
+    endif()
   endif()
 endif()
 
diff -ur qca-2.3.8/plugins/qca-nss/CMakeLists.txt ../qca-2.3.8/plugins/qca-nss/CMakeLists.txt
--- qca-2.3.8/plugins/qca-nss/CMakeLists.txt	2024-01-09 12:12:58.000000000 +0100
+++ ../qca-2.3.8/plugins/qca-nss/CMakeLists.txt	2024-03-14 05:01:25.871900300 +0100
@@ -1,8 +1,11 @@
-find_package(PkgConfig REQUIRED)
-if(WITH_nss_PLUGIN STREQUAL "yes")
-  pkg_check_modules(NSS REQUIRED IMPORTED_TARGET nss)
-else()
-  pkg_check_modules(NSS IMPORTED_TARGET nss)
+find_package(PkgConfig)
+
+if(PKG_CONFIG_FOUND)
+  if(WITH_nss_PLUGIN STREQUAL "yes")
+    pkg_check_modules(NSS REQUIRED IMPORTED_TARGET nss)
+  else()
+    pkg_check_modules(NSS IMPORTED_TARGET nss)
+  endif()
 endif()
 
 if(NSS_FOUND)
diff -ur qca-2.3.8/src/qca_plugin.cpp ../qca-2.3.8/src/qca_plugin.cpp
--- qca-2.3.8/src/qca_plugin.cpp	2024-01-09 12:12:58.000000000 +0100
+++ ../qca-2.3.8/src/qca_plugin.cpp	2024-03-14 05:09:44.894396600 +0100
@@ -492,7 +492,10 @@
     while (!providerItemList.isEmpty()) {
         ProviderItem *i    = providerItemList.first();
         const QString name = i->p->name();
+#if !defined(_MSC_VER)
+	// workaround: crashes for some reason on windows
         delete i;
+#endif
         providerItemList.removeFirst();
         providerList.removeFirst();
 
